tests/unittests/Foundation/NSDecimalNumberTests.mm

```diff
@@ -0,0 +1,332 @@
+//******************************************************************************
```

Comment:
You should probably add some windows-only unit tests at the level of your internal helpers - they're fairly complex, and testing only at the API level is going to obscure a lot of errors/edge cases.